packages/devtools_app/lib/src/screens/inspector/inspector_data_models.dart

```diff
@@ -429,8 +444,8 @@ class FlexLayoutProperties extends LayoutProperties {
 
   String get type => direction.flexType;
 
-  num get totalFlex {
-    if (children?.isEmpty ?? true) return 0;
+  num? get totalFlex {
```

Comment:
return int instead of num

Reply:
in one of descendants it can return double:
https://github.com/flutter/devtools/blob/f7e30d461ac79d0090bfe780e4f5399ee8978097/packages/devtools_app/lib/src/screens/inspector/layout_explorer/flex/utils.dart#L138